WordPressKit/WPStatsServiceRemote.m

```diff
@@ -34,19 +34,20 @@ @interface WPStatsServiceRemote ()
 
 @implementation WPStatsServiceRemote
 
-- (instancetype)initWithOAuth2Token:(NSString *)oauth2Token siteId:(NSNumber *)siteId andSiteTimeZone:(NSTimeZone *)timeZone
+- (instancetype)initWithOAuth2Token:(NSString *)oauth2Token siteId:(NSNumber *)siteId andSiteTimeZone:(NSTimeZone *)timeZone baseUrlString:(NSString *)baseUrlString
 {
     NSParameterAssert(oauth2Token.length > 0);
     NSParameterAssert(siteId != nil);
     NSParameterAssert(timeZone != nil);
+    NSParameterAssert(baseUrlString != nil);
     
     self = [super init];
     if (self) {
         _oauth2Token = oauth2Token;
         _siteId = siteId;
         _siteTimeZone = timeZone;
-        _sitesPathPrefix = [NSString stringWithFormat:@"%@/sites/%@", WordPressComApiClientEndpointURL, _siteId];
-        _statsPathPrefix = [NSString stringWithFormat:@"%@/sites/%@/stats", WordPressComApiClientEndpointURL, _siteId];
+        _sitesPathPrefix = [NSString stringWithFormat:@"%@/rest/v1.1/sites/%@", baseUrlString, _siteId];
```

Comment:
I am just getting myself familiar with the way we do API versioning so Im serious, is it possible to pass the version (as enum or something)?

Reply:
So in almost every other case a subclass of `ServiceRemoteWordPressComREST` is used which has a method, `pathForEndpoint:withVersion:` to do exactly this.

I'm not sure why `WPStatsServiceRemote` is an exception and does not subclass this. I have made a small update to match more closely what is there now (the oath is in a constant).